Makefile

```diff
@@ -44,8 +44,14 @@ build-api-docs:
 			-template-dir /repo/hack/api-docs/api-docs-template \
 			-out-file /repo/docs/api-docs.md"
 
+helm-unittest:
+	helm unittest charts/spark-operator-chart --strict
+
+helm-lint:
+	docker run -it --rm --workdir /workspace --volume $(PWD):/workspace quay.io/helmpack/chart-testing:latest ct lint
```

Comment:
Curious why you added `-it` which requires a terminal?

Reply:
Never mind, I see you completely replace this.

Reply:
> Curious why you added `-it` which requires a terminal?

This is a mistake and I have fixed that in #1967.